doc/book/cookbook/debug-toolbars.md

```diff
@@ -74,11 +74,9 @@ return [
         ],
     ],
     'middleware_pipeline' => [
-        'pre_routing' => [
-            [
-                'middleware' => [
-                    PhpDebugBarMiddleware::class,
-                ],
+        [
+            'middleware' => [
+                PhpDebugBarMiddleware::class,
             ],
```

Comment:
The latest patch also adds a `priority` key, and has the above example updated to specify priority 1000 (meaning prior to routing).